smallcloudai/refact-chat-js
# Pin buttons on same line

# Pull Request Title


## Description

Places the markdown buttons on the same level as the other buttons for the pin message

## Type of change

This means that the markdown component won't have any buttons, those will be handled in the title area.

## How to Test


1. Ask a question that results in a 📍
2. The copy and diff buttons should still work (event though they are in a different place).


## Screenshots (if applicable)



![Screenshot description](url/to/screenshot.png)

## Checklist




## Linked Issues



## Additional Notes

I've disabled the buttons when they can't be used because conditionally rendering them removing them caused some odd layout issues
